src/services/account/useInvoice.ts

```diff
@@ -0,0 +1,99 @@
+import { useQuery } from '@tanstack/react-query'
+import { z } from 'zod'
+
+import Api from 'shared/api'
+import { NetworkErrorObject } from 'shared/api/helpers'
+
+import { InvoiceSchema } from './useInvoices'
+
+const query = `
+query Invoice($owner: String!, $id: String!) {
+  owner(username: $owner) {
+    invoice(invoice_id: $id) {
```

Comment:
Can we have a follow up to covert this to camel case? It's in line with other inputs we have for GQL. eg: `isPublic`

Reply:
Updated! Confirmed when running locally it still works
![Uploading Screenshot 2024-06-10 at 8.50.26 AM.png…]()